### PR TITLE
Pet 23 k6 action

### PR DESCRIPTION
**Pull request by glothriel**

### CODE REVIEW

**Comment by glothriel** [comment]
All the `pet-23-k6-action` will be changed to `master` before the merge

File: benchmark/report/script/report_processor.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    return {"dot": dot, "arrow": arrow}
+
+
+def compare_logic_from_str(logic_name):
+    if logic_name == "higher_is_better":
+        logic = 1
+    elif logic_name == "lower_is_better":
+        logic = -1
+    else:
+        raise Exception("Unhandled coparison logic: " + logic_name)
```
Comment (wronan) [inline benchmark/report/script/report_processor.py:89]:
```suggestion
        raise Exception("Unhandled comparison logic: " + logic_name)
```

File: benchmark/report/script/cli.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import argparse
+
+import report_processor
+
+INDEX_DIRECTORY = ".index"
```
Comment (wronan) [inline benchmark/report/script/cli.py:5]:
```suggestion
```
This seems to not be used.

File: benchmark/report/script/cli.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        epilog="""A program, that synchronizes files between two directories, using a configuration file."""
```
Comment (wronan) [inline benchmark/report/script/cli.py:9]: Incorrect description.
```suggestion
        epilog="""K6 benchmark and reporting automation"""
```

**Review by wronan — APPROVED** [review]

**Pull request merged**